cloud_modules/aws.yml

```diff
@@ -6,9 +6,13 @@
 
   tasks:
 
-    - name: Provision a set of instances (credential_type=aws)
-      ec2:
-         state: "{{ state | default('absent') }}"
-         instance_ids: ["8ut0o26a1g"]  # should not match anything
+    - name: Remove this specific tag from instance (credential_type=aws)
```

Comment:
> Remove this specific tag from instance (credential_type=aws)

if `state` is set to `present` wouldn't this be adding a tag? It seems like the task can both add or remove tags. So maybe `Add / remove specific tag..`?

Reply:
Otherwise, LGTM!

Reply:
yes, reworded, thanks.